docs/model/logic/location.md

```diff
@@ -0,0 +1,72 @@
+# Location
+
+farmOS can track the physical location of Assets over time. This includes map
+geometry as well as relation to other Assets.
+
+## Fixed Assets
+
+Some Assets are *fixed* in location. The geometry of these Assets is
+*intrinsic* to the Asset itself.
+
+Land is an example of an Asset with a fixed location. Although geography
+changes over time, farmOS models land in discreet period parcels with a defined
+geometry.
```

Comment:
* "models land in discreet period parcels with a defined geometry" seems like a pretty confusing way to say this...
* `%s/discreet/discrete/`

Reply:
Ha! I suppose it is confusing. I was particularly pleased with that phrase when I came up with it, to be honest. ;-)

Open to suggestions!

Reply:
Maybe something like;

> "Land is an example of an Asset with a fixed location. Although geography changes over time, land assets in farmOS should be considered a snapshot of the physical land they represent at the current time. This is useful because it provides a simplified model for referring to a logical piece of land and its geometry."

Reply:
I like that! Thanks! :-)